tools/clang/unittests/HLSL/ShaderOpTest.cpp

```diff
@@ -418,6 +418,64 @@ void ShaderOpTest::CreatePipelineState() {
     InitByteCode(&CDesc.CS, pCS);
     CHECK_HR(m_pDevice->CreateComputePipelineState(&CDesc, IID_PPV_ARGS(&m_pPSO)));
   }
+  // Wakanda technology, needs vibranium to work
+#if WDK_NTDDI_VERSION >= NTDDI_WIN10_VB
```

Comment:
If the SDK/DDK is older, `NTDDI_WIN10_VB` would be undefined and resolve to zero, thus passing the comparison when not intended, right?
The fix might need to be some fallback definition in case it's not defined.
```
#ifndef NTDDI_WIN10_VB
#define NTDDI_WIN10_VB 0x0A000008
#endif
```

Reply:
ah yes. I remember now thinking about that, but not looping back to address it. Thanks!